app/mailers/concerns/course_mailer.rb

```diff
@@ -56,9 +56,22 @@ def placeholder_event_link
       group_event_url(group_id: @course.group_ids.first, id: @course.id))
   end
 
+  def placeholder_event_start
+    @course.start_at&.to_date&.then { |date| I18n.l(date) }
+  end
+
+  def placeholder_event_finish
+    @course.finish_at&.to_date&.then { |date| I18n.l(date) }
+  end
```

Comment:
Der duplizierte Code könnte gut in eine eigene Methode ausgelagert werden und wäre damit auch etwas einfacher:

```suggestion
  def placeholder_event_finish
    format_date(@course.finish_at)
  end
  
  def format_date(date)
    return unless date
    
    I18n.l(date.to_date)
  end
```